### DISCUSSION TITLE
MAC电脑启动节点starcoin前，配置环境的时候出错，求老师指点

### DISCUSSION THREAD

**Discussion by Cuteprob** [NonceGeek/Move-Camp#172] [Q&A] [answered]
小白一个，按照starcoin节点安装教程，下载最新版本starcoin-1.12.6-alpha，copy到/usr/local/bin/，执行 ./scripts/dev_setup.sh命令的时候，出现如下错误
![image](https://user-images.githubusercontent.com/32025606/199266576-aa2dd60e-5c37-425e-81e5-d46e71576042.png)

**Comment by zhaoge5859** [comment]
用docker吧，我也是按教程操作多遍没成功，直接下载docker 的应用程序 非常方便

**Comment by Demian101** [comment, marked as answer]
我之前写过一个详细的 Starcoin Setup，
不放按照这个试试？ 
[StarCoin Setup.pdf](https://github.com/NonceGeek/Web3-dApp-Camp/files/9915230/StarCoin.Setup.pdf)
